File: play-2.5.x/K-Modules/kmodules/K-Base/app/models/K/EbeanConfig/MyServerConfigStartup.java
package models.K.EbeanConfig;

import com.avaje.ebean.config.ServerConfig;
import com.avaje.ebean.event.ServerConfigStartup;
import play.Logger;

/**
 * Created by kk on 14/11/28.
 */
public class MyServerConfigStartup implements ServerConfigStartup {
    @Override
    public void onStart(ServerConfig serverConfig) {
        Logger.debug("He, MyServerConfigStartup is called...");
        serverConfig.setEncryptKeyManager(new MysqlEncryptKeyManager());
    }
}

File: play-kotlin-2.5.x/PlayKotlin/kmodules/K-Base/app/models/K/EbeanConfig/MyServerConfigStartup.kt
package models.K.EbeanConfig

import K.Common.Helper
import com.avaje.ebean.config.ServerConfig
import com.avaje.ebean.event.ServerConfigStartup

/**
 * Created by kk on 14/11/28.
 */
class MyServerConfigStartup : ServerConfigStartup {
    override fun onStart(serverConfig: ServerConfig) {
        Helper.DLog("He, MyServerConfigStartup is called...")
        serverConfig.encryptKeyManager = MysqlEncryptKeyManager()
    }
}
